code/datums/ruins/lavaland.dm

```diff
@@ -123,6 +123,22 @@
 	cost = 0
 	allow_duplicates = FALSE
 
+/datum/map_template/ruin/lavaland/blood_drunk_miner
+	name = "Blood-Drunk Miner"
+	id = "blooddrunk"
+	description = "A strange arrangement of stone tiles and an insane, beastly miner contemplating them."
+	suffix = "lavaland_surface_blooddrunk1.dmm"
+	cost = 0
```

Comment:
What is the point of saying "if at all" when you're making a PR that will make it always spawn?